bin/credentials-helper

```diff
@@ -12,12 +11,9 @@ var usageStatistics = require('../src/usage-statistics');
 
 process.on('uncaughtException', function(err) {
   console.error(err.stack.toString());
-  async.parallel([
-    bugtracker.notify.bind(bugtracker, err, 'credentials-helper'),
-    usageStatistics.addEvent.bind(usageStatistics, 'credentials-helper-exception')
-  ], function() {
-    process.exit();
-  });
+  bugtracker.notify(err, 'credentials-helper');
```

Comment:
This code isn't equivalent; the old code waits for bugtracker and usage statistics to finish before calling process.exit (which immediately terminates the process and doesn't wait for remaining tasks afaik). Not terribly important but want to make sure the implications of the change are known